organizations/serializers.py

```diff
@@ -10,9 +13,32 @@
 # pylint: disable=too-few-public-methods
 class OrganizationSerializer(serializers.ModelSerializer):
     """ Serializes the Organization object."""
+    logo_url = serializers.CharField(write_only=True, required=False)
+
     class Meta(object):  # pylint: disable=missing-docstring
         model = models.Organization
-        fields = '__all__'
+        fields = ('id', 'created', 'modified', 'name', 'short_name', 'description', 'logo',
+                  'active', 'logo_url',)
+
+    def update_logo(self, obj, logo_url):  # pylint: disable=missing-docstring
+        logo = requests.get(logo_url)
```

Comment:
Unconditionally getting and reading a file from the internet always makes me nervous. What is the security surface here...? (Hopefully it won't matter much, if we authenticate user has permissions, but still...)

Denial of service from a big file? Maybe a file that is malformed in a way that... would... I dunno, I assume ContentFile or ImageField would choke on a crafted file for us, since we're parsing it ourselves.